src/Waives.Pipelines/Pipeline.cs

```diff
@@ -326,21 +332,21 @@ void OnDocumentException(Exception exception, Document document)
                 }
             }
 
-            Func<Document, Task<WaivesDocument>> docCreator = async d =>
+            Func<Document, CancellationToken, Task<WaivesDocument>> docCreator = async (d, ct) =>
             {
                 _logger.Info("Started processing '{DocumentSourceId}'", d.SourceId);
 
                 var httpDocument = await _documentFactory
-                    .CreateDocumentAsync(d).ConfigureAwait(false);
+                    .CreateDocumentAsync(d, ct).ConfigureAwait(false);
 
                 return new WaivesDocument(d, httpDocument);
             };
 
-            Func<WaivesDocument, Task> docDeleter = async d =>
+            Func<WaivesDocument, CancellationToken, Task> docDeleter = async (d, ct) =>
             {
                 try
                 {
-                    await d.HttpDocument.DeleteAsync().ConfigureAwait(false);
+                    await d.HttpDocument.DeleteAsync(ct).ConfigureAwait(false);
```

Comment:
Same comment here - I don't think these requests should be cancelled, since they should probably run regardless.